cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java

```diff
@@ -52,6 +55,20 @@ public static void CsvLoaderTest() throws IOException {
         loader.load(cube, schemaFileName, dimFileName, dataFileName, cubeRepo);
     }
 
+    @Test(expectedExceptions = { IOException.class },priority = 10, enabled = false)

```

Comment:
We should add logic to check the existence of these files between lines 43-46 in CoolLoader.java

Reply:
added another separate test case for csv loader